common/src/main/scala/uk/ac/wellcome/models/UnifiedItem.scala

```diff
@@ -11,10 +11,20 @@ case class SourceIdentifier(source: String, sourceId: String, value: String)
 
 case class IdentifiedUnifiedItem(canonicalId: String, unifiedItem: UnifiedItem)
 
-/** A representation of an item in our ontology, without a canonical identifier */
-case class UnifiedItem(identifiers: List[SourceIdentifier],
-                       label: String,
-                       accessStatus: Option[String] = None)
+/** A representation of an item in our ontology, without a
+ *  canonical identifier.
+ */
+case class UnifiedItem(
+  identifiers: List[SourceIdentifier],
+  label: String,
+  description: Option[String] = None,
+  lettering: Option[String] = None,
+  hasCreatedDate: Option[String] = None,
```

Comment:
What's the plan for these? Are you going to fill these out in this PR? Has an impact on how we do the first version of the mapping as @alicefuzier and I may want to simply make these strings for now, (rather than the `Period` and `Agent` types we discussed).